add_events.py

```diff
@@ -112,8 +112,12 @@ def main():
                 replaceHour += int(startHour)
             event = {}
             # Currently there are no labs in `subjects.json`
-            if (data[day][time][0] in subjects.keys()):
-                event['summary'] = "Class of " + subjects[data[day][time][0]].title()
+            # if (data[day][time][0] in subjects.keys()):
+            if (data[day][time][2] < 3):
+                if (data[day][time][0] in subjects.keys()):
+                    event['summary'] = "Class of " + subjects[data[day][time][0]].title()
+                else: 
+                    event['summary'] = "Class of " + data[day][time][0]                       
             else:
                 event['summary'] = "Lab of " + data[day][time][0]
```

Comment:
Can you change it according to [this](https://github.com/metakgp/gyft/blob/592174562096556424fb8361922a744ab511dc77/add_events.py#L118).